### DISCUSSION TITLE
How to overwrite TLS cipher suites during runtime?

### DISCUSSION THREAD

**Discussion by Sidhartha-Biswal** [jaegertracing/jaeger#5471] [Q&A]
The TLS cipher suites for collector http interface can be configured with the --collector.http.tls.cipher-suites CLI flag.
Comma-separated list of cipher suites for the server, values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants).

Hi @yurishkuro,

Can the TLS cipher suites be overwritten in the Jaeger code during runtime?

**Comment by Sidhartha-Biswal** [comment]
Hi @yurishkuro,

Does Jaeger support overwriting of cipher suites during runtime?

**Comment by yurishkuro** [comment]
no

**Reply by Sidhartha-Biswal** [reply]
Hi @yurishkuro,

Thanks for the confirmation.

**Comment by Sidhartha-Biswal** [comment]
Hi @yurishkuro,

May I know your view on the runtime overwriting of the cipher suites? Is it a valid use case?

If yes, do you have any plan to implement this in the future?

Thanks

**Comment by yurishkuro** [comment]
You can restart the server to pick up new settings. Go TLS does not support swapping cipher suites.

**Reply by Sidhartha-Biswal** [reply]
thanks @yurishkuro
